guides/common/modules/proc_preparing-for-disaster-recovery-with-active-and-passive-project-server-with-external-storage.adoc

```diff
@@ -0,0 +1,34 @@
+[id="preparing-for-disaster-recovery-with-active-and-passive-project-server-with-external-storage"]
+= Preparing for disaster recovery with active and passive {ProjectServer} with external storage
+
+Create your passive {ProjectServer} as a clone of your active {ProjectServer}.
+Make sure that both servers connect to the database content on your shared storage.
+
+.Procedure
+. Replicate the `/var/lib/pulp` and `{postgresql-lib-dir}` directories from the active {ProjectServer} to your shared storage.
+. Clone your active {ProjectServer}.
+See xref:cloning_satellite_server[].
```

Comment:
I just realized: the cloning process involves a DB-restore, which is probably not what people want.
What would probably work is: clone with `/var/lib/pgsql` being local, then turn things off and replace it with shared storage.
But that's fragile :/

Reply:
When you say that it's "fragile", you're actually kind of saying that that's probably not what we want either :) So what would be a good solution?

Reply:
Yeah… The deployment consists of three things: packages (rpms), config (/etc) and data (/var/lib/). To get a working deployment you need all of them. As the data is quickly moving, we place it on shared storage so it can be picked up from various places. The packages and config are not, so we can keep them on "traditional" storage, but they still need to match.

The easiest way to create a matching set is a "clone" (in the general meaning of the term), but satellite-clone does more than cloning rpms/configs, it also does data.

How we can get to a good clone of the slowly moving parts only: I need to think more about it :D

Reply:
I created issue https://github.com/theforeman/foreman-documentation/issues/3714 where we can track this. According to https://github.com/theforeman/foreman-documentation/pull/3679#pullrequestreview-2662882118, this open item doesn't block merging.